### ISSUE TITLE
List of PHIPO_EXT terms used in annotations

### ISSUE THREAD

**Issue by CuzickA** [PHI-base/phipo_ext#1]
Hi @jseager7 please could you provide me with a list of the PHIPO_EXT terms in annotations. Thanks.

I would like to obsolete several exploratory terms

**Comment by jseager7** [comment]
@CuzickA Here's all the PHIPO_EXT terms that have been used in curation sessions, current as of 18 May:

Term ID|Name|Comment
---|---|---
PHIPO_EXT:0000012|compatible interaction, recognizable pathogen effector present, functional host resistance gene absent|keep
PHIPO_EXT:0000013|compatible interaction, recognizable pathogen effector absent, functional host resistance gene present|keep
PHIPO_EXT:0000014|compatible interaction, recognizable pathogen effector present, compromised host resistance gene|keep
PHIPO_EXT:0000018|incompatible interaction, recognizable pathogen effector present, functional host resistance gene present|keep
PHIPO_EXT:0000019|incompatible interaction, recognizable pathogen effector present, gain of functional host resistance gene|keep
PHIPO_EXT:0000020|incompatible interaction, recognizable pathogen effector present, compromised host resistance gene|need to obsolete, past usage in atr1/RPP1
PHIPO_EXT:0000030|incompatible interaction, recognizable pathogen necrotrophic effector present, host susceptibility locus compromised
PHIPO_EXT:0000031|compatible interaction, recognizable pathogen necrotrophic effector present, functional host susceptibility locus present
PHIPO_EXT:0000032|incompatible interaction, recognizable pathogen necrotrophic effector present, host susceptibility locus absent
PHIPO_EXT:0000033|compatible interaction, recognizable pathogen necrotrophic effector present, gain of host susceptibility locus
PHIPO_EXT:0000034|incompatible interaction, recognizable pathogen necrotrophic effector absent, host susceptibility locus present
PHIPO_EXT:0000035|compatible interaction, gain of recognizable pathogen necrotrophic effector, host susceptibility locus present
PHIPO_EXT:0000036|incompatible interaction, gain of recognizable pathogen necrotrophic effector, host susceptibility locus compromised
PHIPO_EXT:0000038|compatible interaction, compromised recognizable pathogen necrotrophic effector, host susceptibility locus present
PHIPO_EXT:0000041|incompatible interaction, compromised recognizable pathogen necrotrophic effector, host susceptibility locus present
PHIPO_EXT:0000042|incompatible interaction, gain of recognizable pathogen effector, functional host resistance gene present|keep
PHIPO_EXT:0000044|compatible interaction, recognizable pathogen effector absent, functional host resistance gene compromised|keep
PHIPO_EXT:0000045|compatible interaction, compromised recognizable pathogen effector, functional host resistance gene present|keep
PHIPO_EXT:0000046|metagenotype outcome overcome by external condition|keep

**Comment by CuzickA** [comment]
@jseager7 please confirm which session this term was used in?

PHIPO_EXT:0000020 | incompatible interaction, recognizable pathogen effector present, compromised host resistance gene

I deleted it from https://github.com/PHI-base/curation/issues/19 this morning so perhaps it is only used in a test session?


Note to self: make all other GfG compound terms in PHIPO_EXT obsolete using PHIPO_EXT term creator spreadsheet (if I've not already done so).

Next: check inverse GfG PHIPO_EXT terms

**Comment by jseager7** [comment]
@CuzickA According to the export, PHIPO_EXT:0000020 is used only once, in [PMID:20601497](https://canto.phi-base.org/curs/f41d30e250b8f9b0/ro/). However, I can't find it in that session, so I can only assume it's already been replaced since the export was last made.

**Comment by CuzickA** [comment]
> @CuzickA According to the export, PHIPO_EXT:0000020 is used only once, in [PMID:20601497](https://canto.phi-base.org/curs/f41d30e250b8f9b0/ro/). However, I can't find it in that session, so I can only assume it's already been replaced since the export was last made.

Thanks

**Comment by CuzickA** [comment]
@jseager7 I have checked and obsoleted the old PHIPO_EXT terms for the GfG AE set. Please could you load these changes so that I can check that only the required terms are available in PHI-Canto. Thanks.

I'll work on the inverse GfG terms next.
